main/resource-access.bs

```diff
@@ -142,19 +139,151 @@ through POST, using the HTTP `Slug` header as defined in [[!RFC5023]].
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)]
+
+For resource representations having an algorithm to perform an append
+operation, servers MUST update resources for HTTP `POST` requests. Servers
+MUST perform RDF merge [[!RDF11-MT]] on RDF sources [[!RDF-CONCEPTS]].
+[[Source](https://github.com/solid/specification/issues/108#issuecomment-573655070)]
+
+When a `PUT` or `PATCH` method request targets an auxiliary resource, the
+server MUST create or update it. When a `POST` method request with the `Slug`
+header targets an auxiliary resource, the server `MUST` respond with the `403`
+status code and response body describing the error.
+[[Source](https://github.com/solid/specification/issues/42#issuecomment-616688848)]
+
+Servers SHOULD NOT allow HTTP `POST`, `PUT` and `PATCH` to update a
+container's containment triples; if the server receives such a request, it
+SHOULD respond with a `409` status code.
+[[Source](https://github.com/solid/specification/issues/40#issuecomment-573358652)]
+
+Clients MUST use the HTTP `If-None-Match` header with a value of `"*"` to
+prevent an unsafe method from updating an existing representation of the
+target resource when creation of a new resource is intended.
+[[Source](https://github.com/solid/specification/issues/108#issuecomment-567272797)]
+[[Source](https://github.com/solid/specification/issues/40#issuecomment-566995240)]
+
+Clients MUST use the HTTP `If-Match` header and HTTP `ETags` to ensure it
+isn't modifying a resource that has changed since the client last retrieved
+its representation. Servers MUST require the HTTP `If-Match` header and HTTP
+`ETags` to detect collisions. Servers MUST respond with status code `412` if
+`ETags` fail to match when there are no other errors with the request
+[[!RFC7232]]. Servers MUST respond with status code `428` and response body
+describing the error when the absence of a precondition is the only reason for
+rejecting the request [[!RFC6585]].
+[[Source](https://github.com/solid/specification/issues/60)]
+[[Source](https://github.com/solid/specification/issues/33)]
+
+When using Web Access Control ([[#wac]]):
+
+To create or update an ACL resource (see [[#ar-wac]]), an `acl:agent` MUST
+have `acl:Control` privileges per the ACL inheritance algorithm on the
+resource directly associated with it.
+[[Source](https://github.com/solid/specification/issues/42#issuecomment-616688848)]
 
-Clients who want to create hierarchical containers MUST use PUT and PATCH
-requests.
+<em>This section is non-normative.</em>
 
-[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)].
+While it is technically possible to auxiliary resources using the HTTP `POST`
+method with the `Slug` header, it allows varying implementations due to its
+non-deterministic resource naming.
+
+### Deleting Resources ### {#delete}
+
+Servers MUST support the HTTP `DELETE` method [[!RFC7231]].
```

Comment:
Idem as above; read-only Solid servers are a thing.

Reply:
https://github.com/solid/specification/pull/188/commits/6eb9436bbc43318dcab1628bad16b9e3888fa2a6

Reply:
https://github.com/solid/specification/pull/188/commits/09cc77a20cab781177e18e4b970cd4e3426ee5d0 should ties this up.